modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -75,7 +75,7 @@ def self.evss_id_by_token(token)
 
     def self.get_by_id_or_evss_id(id)
       if id.to_s.include?('-')
-        find(id)
+        find_by(id: id)
```

Comment:
`find` raises a `RecordNotFound` error if there's not a matching ID. We do that with our own error messaging in the 2 calls to this method, so switching to `find_by` (returns `nil` if not found) and delegating the errors to the methods

Reply:
i wonder if this breaks expectations in v0 or v1 ?

Reply:
v1 was redundantly checking/raising 404s before. [I added that in v0, as well,](https://github.com/department-of-veterans-affairs/vets-api/pull/7455/commits/6572305515fea4df12be44c9bb758d287a7b4769) so I think we're good now